setup.py

```diff
@@ -53,7 +54,7 @@ def text_from_file(path):
         "grpcio",  # https://github.com/googleapis/google-cloud-python/issues/6259
         "incremental",
         "jinja2",
-        "mozanalysis",
+        # "mozanalysis",
```

Comment:
Todo: uncomment

I'm currently using my locally built version of https://github.com/mozilla/mozanalysis/pull/125 for testing.